### PR TITLE
SPOPT-1801 - formatting VPP CLI response

### PR DESCRIPTION
**Pull request by mpundlik**

### CODE REVIEW

File: docker/dev_vpp_agent/build-agent.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ curl https://glide.sh/get | sh
 # checkout agent code
 mkdir -p $GOPATH/src/github.com/ligato
 cd $GOPATH/src/github.com/ligato
-git clone https://github.com/ligato/vpp-agent
+git clone https://github.com/mpundlik/vpp-agent
```
Comment (ondrej-fabry) [inline docker/dev_vpp_agent/build-agent.sh]: Why did you need this committed to this pull request?

Comment (mpundlik) [reply docker/dev_vpp_agent/build-agent.sh]: not needed, removed

**Comment by mpundlik** [comment]
Will be creating a new PR
